src/main/java/net/dv8tion/jda/api/requests/restaction/TestEntitlementCreateAction.java

```diff
@@ -0,0 +1,86 @@
+package net.dv8tion.jda.api.requests.restaction;
```

Comment:
We have a copyright header on all files, you need to add that to your new classes